### PR TITLE
Support signing unsigned txs using ledger accounts

### PR DESCRIPTION
**Pull request by Lbqds**
Screenshot 2024-10-24 at 09 14 09

Screenshot 2024-10-24 at 08 30 25

### CODE REVIEW

File: packages/extension/src/background/actionHandlers.ts [hunk 1]
```diff
@@ -256,3 +277,19 @@ export const handleActionRejection = async (
       assertNever(action)
   }
 }
+
+// TODO: use the function from web3-sdk
```
Comment (Lbqds) [inline packages/extension/src/background/actionHandlers.ts:281]: Once [this PR](https://github.com/alephium/alephium-web3/pull/437) is merged, we can use the `TransactionBuilder.buildUnsignedTx` function from the SDK.

**Review by h0ngcha0 — APPROVED** [review]
Nice refactoring, LGTM :+1:

File: packages/extension/src/ui/features/actions/LedgerStatus.tsx
Comment (polarker) [inline packages/extension/src/ui/features/actions/LedgerStatus.tsx:1]: Let's put JSX code in this file and move the hooks to another file as they can be reused by other wallets.

**Review by polarker — COMMENTED** [review]
Let's also change `Sign Unsigned TX` as `Sign Raw TX`, which might be more user-friendly.

**Review by polarker — APPROVED** [review]
LGTM 👍

**Pull request merged**